ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -8,6 +8,7 @@
       - '{name}-{test_name}-for-period':
           test_name: job-updater
           node: 'antrea-test-node'
+          jenkins_location: 'cloud'
```

Comment:
We have already defined jenkins_location in builder-job-updater, may I know why add one more item?